src/Simplex/Chat/Store/Connections.hs

```diff
@@ -37,6 +39,49 @@ import qualified Simplex.Messaging.Agent.Store.SQLite.DB as DB
 import Simplex.Messaging.Crypto.Ratchet (PQSupport)
 import Simplex.Messaging.Util (eitherToMaybe)
 
+getConnectionEntityId :: DB.Connection -> AgentConnId -> ExceptT StoreError IO ConnectionEntityId
```

Comment:
couldn't split getConnectionEntity, because its first step is reading connection by id, and it also could be updated so it should be read after taking lock. also it's used in other places as well